docs/installation/security.rst

```diff
@@ -0,0 +1,218 @@
+.. _installation_security:
+
+===================================
+Securing an Open Forms installation
+===================================
+
+The default configuration options of Open Forms aim to provide a secure setup which
+enables all functionalities. We provide sufficient room to tighten this security if
+desired, which may come at the cost of disabling certain features, or to loosen the
+security for quick-and-dirty exploration installations.
+
+We recommend reading and understanding this document before performing penetration
+tests. The contents have been composed based on pentest reports we have received.
+
+.. warning:: Do not blindly take the configuration from the ``docker-compose.yml`` file
+   - it is not suitable for real deployments, only for exploration.
+
+Throughout this document, certain configuration options will be referenced. The full
+list of configuration options and how to apply them is documented in the
+:ref:`installation_environment_config`.
+
+Cross-Origin Resource Sharing (CORS)
+====================================
+
+Open Forms operates in CORS-mode when you embed forms on third-party domains (such as
+product and services pages) via a CMS or other similar solutions. The embedded form
+makes calls to the domain where Open Forms is deployed and exposing the API endpoints.
+
+If you don't plan to use this embedding feature, you don't need CORS.
```

Comment:
If you want to embed forms on your own trusted domain (like your own website), then you need to configure CORS appropriatly. (please link to CORS at Mozilla + link to embedding)

It's good to understand that CORS is not a securty mechanism but rather a feature to allow exceptions, such as embedding Open Forms on your own website.

Reply:
Explain third party domain, trusted domain